CODE_OF_CONDUCT.md

```diff
@@ -15,7 +15,7 @@ diverse, inclusive, and healthy community.
 ## Our Standards
 
 Examples of behavior that contributes to a positive environment for our
-community include:
+community includes:
```

Comment:
issue: adding a trailing _-s_ in “includes” is a mistake

In Present Simple, plural nouns (“examples”) drop the trailing _-s_ in the associated verb (“include”). The correct form is “Examples [...] include”, as it currently is.